docs/book/src/clusterctl/overview.md

```diff
@@ -151,6 +151,11 @@ For more information about prerequisites, credentials management, or permissions
 
 Please visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/getting-started.md).
 
+{{#/tab }}
+{{#tab Bare Metal}}
```

Comment:
I did not check the preview, but I'm pretty sure the new tab should be added to the list of tabs at line 77

Reply:
fixed, thank you